### PR TITLE
Chore/refresh from storage

### PR DESCRIPTION
**Pull request by donnapep**
* Add the ability for Storage images to refresh every 15 minutes.
* Update the test structure so that it is similar to the Video Widget.
* Sometimes the call to `storage.go()` fails, and that's because the web component hasn't initialized yet. Add a `polymer-ready` event handler to tell when the component has been registered.
* Update the tests to manually fire the `polymer-ready` event since otherwise it will never trigger.
* Use `rise-storage-mock` to ensure the Storage image has loaded.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees I think we need to start using the `polymer-ready` event in any Widget that uses a web component. Please review. Thx.

File: gulpfile.js [hunk 1]
```diff
@@ -103,27 +104,67 @@
...
+        path.basename += "-url-e2e";
+      }))
+      .pipe(gulp.dest("./src/"));
+  });
+
+  gulp.task("html:e2e:widget:storage", function () {
+    return gulp.src("./src/widget.html")
+      .pipe(htmlreplace({
+        e2egadgets: "../node_modules/widget-tester/mocks/gadget-mocks.js",
+        e2eMockData: ["../node_modules/sinon/pkg/sinon.js", "../node_modules/sinon/pkg/sinon-server-1.12.2.js",
```
Comment (stulees) [inline gulpfile.js]: @donnapep So this actually injects three separate imports? I didn't know it could do that!

**Comment by stulees** [comment]
@donnapep :+1:

File: gulpfile.js (hunk 1 above)
Comment (donnapep) [inline gulpfile.js]: @stulees Yup, I discovered it awhile back when I did some playing with the gulp test tasks. Comes in handy.

**Comment by donnapep** [comment]
@stulees I updated the Storage modal URL and also ran the build against Production. Please review. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**